python/lsst/pipe/tasks/coaddBase.py

```diff
@@ -251,3 +252,36 @@ def subBBoxIter(bbox, subregionSize):
                                    "colShift=%s, rowShift=%s" %
                                    (bbox, subregionSize, colShift, rowShift))
             yield subBBox
+
+
+# Note that this is implemented as a free-floating function to enable reuse in
+# lsst.pipe.tasks.makeWarp and in lsst.drp.tasks.make_psf_matched_warp
+# without creating any relationships between the two classes.
+# This may be converted to a method after makeWarp.py is removed altogether.
```

Comment:
I think you have the ticket number for removing `makeWarp.py`. I’d suggest adding it here.